tfx/experimental/distributed_inference/graphdef_experiments/subgraph_partitioning/graph_partition.py

```diff
@@ -0,0 +1,471 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
Are there any dependencies between `graph_partition.py` and the rest of this PR? If so, consider removing `graph_partition.py` and its test from this PR and moving it to a new PR.

Reply:
Moved into two PRs.

Reply:
Where is the other PR?